src/Microsoft.ML.AutoML/API/MulticlassClassificationExperiment.cs

```diff
@@ -159,7 +159,7 @@ public override ExperimentResult<MulticlassClassificationMetrics> Execute(IDataV
             // split cross validation result according to sample key as well.
             if (rowCount < crossValRowCountThreshold)
             {
-                const int numCrossValFolds = 10;
+                int numCrossValFolds = 10;
```

Comment:
why you have removed the const?

Reply:
There's no specific reason of doing that. I just feel like there's no need to mark this `numCrossValFolds` as const.

Reply:
why you have a variable in the first place then? I mean why you don't just call the method and passing `10`?

Reply:
I didn't notice that,, fixed and Resolved